.azurepipelines/Windows-VS.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     do_non_ci_build: false
     do_non_ci_setup: true
     do_pr_eval: true
-    packages: QemuQ35Pkg
+    packages: QemuQ35Pkg,QemuPkg

```

Comment:
We only run `NO-TARGET` here because the platform builds handle actual compilation.

I don't see compilation for QemuPkg. Is it in CI in your current set of changes?